test/functional/services/remote/webdriver.ts

```diff
@@ -125,6 +125,7 @@ async function attemptToCreateCommand(
         });
         chromeCapabilities.set('unexpectedAlertBehaviour', 'accept');
         chromeCapabilities.set('goog:loggingPrefs', { browser: 'ALL' });
+        chromeCapabilities.set('acceptInsecureCerts', true);
```

Comment:
I use this to ignore insecure certificate errors only. https://github.com/elastic/kibana/pull/68544/files#diff-29364962baee077efdfea89294c62fd8L114-R116 ignores all the cert errors.

Reply:
I remember discussing it with @spalger and we decided not to go this way. I understand it might be needed for a particular test. How about we use a flag here?

Reply:
I'm fine to add a flag, but since it's an ordinary test, we will have to set the flag when running the test locally and on CI. Is it worth effort?

Reply:
How about adding `browser.extra` in `x-pack/test/functional_embedded/config.ts`:
```
browser: {
  extra: {
    acceptInsecureCerts: true,
  }
}
```
that can be passed as the 5th argument here: https://github.com/elastic/kibana/blob/master/test/functional/services/remote/remote.ts#L61-L66

and simply use if statement to set extra browser settings.

It requires changes, but at least we won't run all the functional tests with that option enabled.

Reply:
done